src/config/server.rs

```diff
@@ -20,7 +20,9 @@ pub struct Server {
   pub enable_response_validation: Option<bool>,
   pub global_response_timeout: Option<i64>,
   pub port: Option<u16>,
-  pub proxy: Option<Proxy>,
+  #[serde(default)]
+  #[serde(skip_serializing_if = "is_default")]
```

Comment:
```suggestion
  #[serde(default, skip_serializing_if = "is_default")]
```

You can just do like this